demo/src/test/java/com/example/test/pages/KBBMOOCProject/ButtonLXCoupe2D.java

```diff
@@ -0,0 +1,18 @@
+package com.example.test.pages.KBBMOOCProject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ButtonLXCoupe2D extends KbbBasePage {
+
+    public ButtonLXCoupe2D(WebDriver driver) {
+        super(driver);
+        //TODO Auto-generated constructor stub
+    }
+
+    WebElement LXCoupe2D = keyword.findWebElementByXpath("//a[@class='css-1sw5zct erywcq03']");
```

Comment:
should write the web element locator in separate line then call into a method/function